src/datasources/exchange-api/exchange-api.service.spec.ts

```diff
@@ -60,7 +60,7 @@ describe('ExchangeApi', () => {
     expect(mockCacheFirstDataSource.get).toBeCalledWith(
       new CacheDir('exchange_fiat_codes', ''),
       `${exchangeBaseUri}/symbols`,
-      { params: { access_key: exchangeApiKey } },
+      { headers: { apikey: exchangeApiKey } },
```

Comment:
Unfortunately even the test was testing against the older API version (so the test was wrong too)